hooks/post_checkout

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+git clone https://github.com/HL7-DaVinci/CDS-Library.git server/CDS-Library
```

Comment:
Should we change this line to use `CDS-Library` in the `mcode` space?

Reply:
no, the hooks/post_checkout file is for the DRLS CI/CD pipeline, so we can ignore it since we won't be using that.